tests/unit_tests/test_lib_bfs.h

```diff
@@ -35,14 +35,8 @@ TEST(sharedlibrary, breadthfirstsearch)
 
     int *labels = (int*)malloc(sizeof(int) * graphi->num_nodes);
     labels = (int*)grapho->node_value1;
-    // int node; for (node = 0; node < graphi->num_nodes; ++node)
-    //    printf("Node_ID [%d] : Label [%d]\n", node, labels[node]);
-
     int result[7] = {2147483647, 2147483647, 0, 1, 1, 1, 2};
```

Comment:
@jowens Should I be writing a reference BFS or having hard coded results work? I was looking at how `tinydnn` implemented gtest and for most of their stuff they had a file with manually written results. That's why I went for this. Your thoughts?